apps/create-vesting/src/hooks/useGetHooksInfoVestUserAmount.ts

```diff
@@ -23,7 +23,9 @@ export const useGetHooksInfoVestUserAmount = () => {
 
       if (!context?.account || !cowShedProxy || !amount) return;
 
-      const periodInSeconds = period * scaleToSecondsMapping[periodScale];
+      const periodInSeconds = Math.ceil(
+        period * scaleToSecondsMapping[periodScale],
+      );
```

Comment:
I know that this logic is simple but it seems to repeat multiple times on the code. I would separate it in new file.